### PR TITLE
docs: upgrade sp runbook and joining process

### PR DESCRIPTION
**Pull request by sysvm**
### Description

Upgrade SP runbook and joining process.

### Rationale

N/A

### CODE REVIEW

**Review by KeefeL — APPROVED** [review]

**Comment by maxnodereal** [comment]
We can add a section to test SP:

Test your integration with [Dcellar](dcellar.io)
1. Connect with your own wallet
2.  Bridge some BNB to Greenfield testnet
3. Click “Create a bucket” and choose your own SP
4. Upload some files to your bucket.

File: src/guide/storage-provider/run-book/run-testnet-SP-node.md [hunk 1]
```diff
@@ -67,125 +67,98 @@ cd greenfield-storage-provider/build
...
+Storage = 's3'
+BucketURL = '' # writing to environment variables is recommended
+MaxRetries = 5
+MinRetryDelay = 0
+TLSInsecureSkipVerify = false
+IAMType = 'SA'
+
+[Chain]
+ChainID = '${chain_id}'
+ChainAddress = ['${chain_address}']
```
[the remark this answers is not in the document]
Comment (sysvm) [reply src/guide/storage-provider/run-book/run-testnet-SP-node.md:104]: added

File: src/guide/storage-provider/run-book/run-testnet-SP-node.md [hunk 2]
```diff
@@ -67,125 +67,98 @@ cd greenfield-storage-provider/build
...
+
+[SpAccount]
+SpOperatorAddress = '${sp_operator_address}'
+OperatorPrivateKey = '' # writing to environment variables is recommended
```
[the remark this answers is not in the document]
Comment (sysvm) [reply src/guide/storage-provider/run-book/run-testnet-SP-node.md:108]: This description is fixed.

File: src/guide/storage-provider/run-book/run-testnet-SP-node.md [hunk 3]
```diff
@@ -67,125 +67,98 @@ cd greenfield-storage-provider/build
...
-B = 1000
-# stop serving configuration
-[DiscontinueCfg]
-BucketKeepAliveDays = 7
-# metadata configuration
-[MetadataCfg]
+P2PPrivateKey = '' # writing to environment variables is recommended
+P2PAddress = '0.0.0.0:9933'
+P2PAntAddress = '${load_balance_doamin:port}'
+P2PBootstrap = ['node_id@load_balance_doamin:port'] # p2p node's bootstrap node, format: [node_id1@ip1:port1, node_id2@ip1:port2]
```
[the remark this answers is not in the document]
Comment (sysvm) [reply src/guide/storage-provider/run-book/run-testnet-SP-node.md:135]: Add more descriptions.

**Pull request merged**